pandas/tests/generic/test_generic.py

```diff
@@ -1019,3 +1019,15 @@ def test_pipe_panel(self):
 
             with pytest.raises(ValueError):
                 result = wp.pipe((f, 'y'), x=1, y=1)
+
+    @pytest.mark.parametrize('box', [pd.Series, pd.DataFrame])
+    def test_axis_classmethods(self, box):
+        obj = box()
+        values = list(box._AXIS_NAMES.keys()) + \
```

Comment:
What's the current status on backslash to escape long lines. Someone (@jbrockmendel?) put a lot of effort in to clean these up. Do we have an issue to make these a linting rule?

Reply:
@qwhelan would you mind converting these to use opening and closing parenthesis to wrap the long lines?

Reply:
Done